README.md

```diff
@@ -299,3 +299,17 @@ spec:
           userDataSecret:
             name: windows-user-data
 ```
+### Windows nodes Kubernetes component upgrade
```

Comment:
Upgrading Kubernetes node components on Windows nodes

Reply:
Eh actually I dont know how i feel about repeating `nodes` twice. Use your best judgement here

Reply:
I am keeping it as is for now